src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -528,6 +528,7 @@ def msOutputConsumer(self):
                 #    start using those here so we do not mix with general errors
                 try:
                     pipeLine.run(docOut)
+                    wfCounters += 1
```

Comment:
If the goal of this counter was to count the amount of workflows going through this specific pipeline, then I would suggest moving this line 1 line above. Otherwise we will report only on successful workflows. Just a thought though, it is not a must change.

Perhaps what would be nice - but then it slightly goes beyond the scope of this issue, would be to log something like (L549):
```
            self.logger.info("Successfully processed %d workflows from pipeline: %s", wfCounters, pipeLineName)
            self.logger.info("Failed to process %d workflows from pipeline: %s", wfCountersFail, pipeLineName)
```

Reply:
I am sorry for the confusion. I got confused and thought in order to maintain the current behavior, which counts in `wfCounters` only wfs that have successfully been processed (see [1], that counter is increased only if we do not end up in any break/continue), I had to move this increment into the `try` clause.

I added a new commit that increases the clarity of the counters.

[1] https://github.com/dmwm/WMCore/blob/44d2a8a77a42f885bb99f3c4128b823b4e1929ea/src/python/WMCore/MicroService/MSOutput/MSOutput.py#L546